internal/pkg/model/schema.go

```diff
@@ -48,8 +48,8 @@ type Action struct {
 	// The input type the actions should be routed to.
 	InputType string `json:"input_type,omitempty"`
 
-	// The minimum time (in seconds) provided for an action execution when scheduled by fleet-server.
-	MinimumExecutionDuration int64 `json:"minimum_execution_duration,omitempty"`
+	// The rollout duration (in seconds) provided for an action execution when scheduled by fleet-server.
+	RolloutDurationSeconds int64 `json:"rollout_duration_seconds,omitempty"`
```

Comment:
This file is generated from `model/schema.json`. you need to update that file then run `make update` to update this file.